diff --git a/dist/components/NuTag/Item.vue b/dist/components/NuTag/Item.vue
index 92ee5bf9247d3e4e971e4566175d7181034c2844..a372ef1b2ad3618297e19f2d53d4cb23c609d142 100644
--- a/dist/components/NuTag/Item.vue
+++ b/dist/components/NuTag/Item.vue
@@ -3,7 +3,7 @@ import { useSlots } from 'vue'
 
 const slots = useSlots()
 
-function hasSlot(name: string) {
+function hasSlot(name: 'pre' | 'post') {
   return !!slots[name]
 }
 </script>
